### PR TITLE
Translate "Ruby 3.3.0-preview3 Released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: #2818, #3136

Actual diff is  ece801b1ea4f85915877d1c4e63e1c287269afa0

### CODE REVIEW

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md [hunk 1]
```diff
@@ -1,163 +1,162 @@
...
-### Removed methods
+### 삭제된 메서드
 
-The following deprecated methods are removed.
+폐기 예정이었던 메서드가 삭제됩니다.
 
-### Removed environment variables
+### 삭제된 환경 변수
 
-The following deprecated methods are removed.
```
Comment (riseshia) [inline ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md]: https://github.com/ruby/www.ruby-lang.org/pull/3140

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+        * `Prism.dump(source, filepath)`는 문자열로 직렬화된 AST를 반환합니다.
+        * `Prism.parse_comments(source)`는 주석을 반환합니다.
+* 기여에 관심이 있다면 [Prism 저장소](https://github.com/ruby/prism)에서 직접 풀 리퀘스트나 이슈를 만들 수 있습니다.
+
+## Bison을 Lrama로 대체
+
+* Bison 대신 [Lrama LALR 파서 생성기](https://github.com/yui-knk/lrama)를 사용하도록 변경했습니다. [Feature #19637](https://bugs.ruby-lang.org/issues/19637)
+  * 관심이 있다면 [Ruby 파서의 미래 비전](https://rubykaigi.org/2023/presentations/spikeolaf.html)을 참고하세요.
+  * 유지보수성을 위해 Lrama 내부 구문 분석기가 Racc에서 생성한 LR 구문 분석기로 대체되었습니다.
+  * 매개변수화 규칙 `(?, *, +)`를 지원하며, Ruby의 parse.y에서 사용됩니다.
```
Comment (yous) [inline ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md:31]:
```suggestion
  * 매개변수화 규칙 `(?, *, +)`를 지원하며, Ruby의 parse.y에서 사용될 예정입니다.
```

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+
+## RJIT
+
+* 순수 Ruby JIT 컴파일러 RJIT을 도입하고 MJIT을 대체했습니다.
+  * RJIT은 Unix 플랫폼에서 x86\_64 아키텍처만 지원합니다.
```
Comment (yous) [inline ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md:36]:
```suggestion
  * RJIT은 Unix 플랫폼에서 x86-64 아키텍처만 지원합니다.
```

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+---
+
+{% assign release = site.data.releases | where: "version", "3.3.0-preview3" | first %}
+Ruby {{ release.version }} 릴리스를 알리게 되어 기쁩니다. Ruby 3.3은 Prism이라고 명명된 새 파서를 추가하고, 파서 생성기로 Lrama를 사용하며, RJIT으로 명명된 새로운 순수 Ruby JIT 컴파일러를 추가하고, 특히 YJIT에서 많은 성능 향상이 있습니다.
+
+## Prism
+
+* [Prism 파서](https://github.com/ruby/prism)를 기본 gem에 추가했습니다.
+    * Prism은 Ruby 언어를 위한 이식 가능하고, 에러 내성이 있으며 유지 보수 가능한 재귀 하향 파서입니다.
+* Prism은 운영 환경에서 사용할 준비가 되어있으며 활발하게 유지보수되고 있으므로 Ripper 대신 이용할 수 있습니다.
```
Comment (yous) [inline ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md:17]: '프로덕션' is used below, so let's make this consistent.

```suggestion
* Prism은 프로덕션 환경에서 사용할 준비가 되어있으며 활발하게 유지보수되고 있으므로 Ripper 대신 이용할 수 있습니다.
```

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+* 3.2보다 메모리 사용량이 크게 개선되었습니다.
+    * 컴파일된 코드의 메타데이터가 훨씬 적은 메모리를 사용합니다.
+    * ARM64에서 더 작은 코드를 생성합니다.
+* 3.2보다 컴파일 속도가 약간 개선되었습니다.
+* 실행 시간에 YJIT을 활성화할 수 있는 `RubyVM::YJIT.enable`이 추가됩니다.
+  * 실행 명령의 인수나 환경 변수의 변경없이 YJIT을 실행할 수 있습니다.
+  * 애플리케이션 부팅이 완료된 후에 YJIT를 활성화하는 데에도 사용할 수 있습니다.
+    부팅 시 YJIT를 비활성화한 상태에서 다른 YJIT 옵션을 사용하려는 경우 `--yjit-disable`을 사용할 수 있습니다.
+* 코드 GC를 비활성화하고 `--yjit-exec-mem-size`를 엄격한 제한으로 설정하는 옵션이 추가됩니다.
+  * 유니콘과 포크를 사용하는 서버에서 더 나은 쓰기시 복사 동작을 생성할 수 있습니다.
```
Comment (yous) [inline ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md:64]:
```suggestion
  * unicorn과 포크를 사용하는 서버에서 더 나은 쓰기시 복사 동작을 생성할 수 있습니다.
```

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+* `--yjit-stats`로부터 생성된 `ratio_in_yjit` 통계는 릴리스 빌드에서도 이용 가능하며,
+  특수한 통계나 개발 빌드는 더 이상 필요하지 않습니다.
```
Comment (yous) [inline ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md:66]:
```suggestion
  대부분의 통계에 접근하는 데 특수한 통계나 개발 빌드는 더 이상 필요하지 않습니다.
```

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md [hunk 7]
```diff
@@ -0,0 +1,293 @@
...
+* 종료 추적 옵션이 이제 샘플링을 지원합니다.
+  * `--trace-exits-sample-rate=N`
+* 리눅스 perf로 프로파일링을 용이하게 할 수 있도록 `--yjit-perf`가 추가됩니다.
```
Comment (yous) [inline ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md:69]:
```suggestion
* Linux perf로 프로파일링을 용이하게 할 수 있도록 `--yjit-perf`가 추가됩니다.
```

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md [hunk 8]
```diff
@@ -0,0 +1,293 @@
...
+* 보다 철저한 테스트와 여러 버그 수정
+
+### M:N 스레드 스케쥴러
+
+* M:N 스레드 스케쥴러가 추가됩니다. [[Feature #19842]](https://bugs.ruby-lang.org/issues/19842)
+  * M개의 Ruby 스레드를 N개의 네이티브 스레드(OS 스레드)로 관리하므로 스레드의 생성, 관리 비용이 절감됩니다.
+  * C 확장 호환성에 문제가 발생할 가능성이 있으므로, 주 Ractor에서의 기본 설정은 비활성입니다.
+      * `RUBY_MN_THREADS=1` 환경 변수를 사용해 주 Ractor에서 M:N 스레드를 활성화할 수 있습니다.
+      * M:N 스레드는 주 Ractor가 아닌 Ractor에서 활성화되어 있습니다.
+  * `RUBY_MAX_CPU=n` 환경 변수는 `N`(네이티브 스레드의 최대 개수)의 최대값을 지정합니다. 기본값은 8입니다.
```
Comment (yous) [inline ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md:79]:
```suggestion
  * `RUBY_MAX_CPU=n` 환경 변수는 `N`(네이티브 스레드의 최대 개수)의 최댓값을 지정합니다. 기본값은 8입니다.
```

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md [hunk 9]
```diff
@@ -0,0 +1,293 @@
...
+      * 하나의 Ractor에서는 하나의 Ruby 스레드만 실행 가능하므로 실제로 사용되는 네이티브 스레드의 게수는 `RUBY_MAX_CPU`에서 지정한 숫자이거나 실행중인 Ractor의 개수입니다. 그러므로 (대다수를 차지하는) 단일 Ractor 애플리케이션에서는 하나의 네이티브 스레드를 사용합니다.
```
Comment (yous) [inline ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md:80]:
```suggestion
      * 하나의 Ractor에서는 하나의 Ruby 스레드만 실행 가능하므로 실제로 사용되는 네이티브 스레드의 개수는 `RUBY_MAX_CPU`에서 지정한 숫자나 실행 중인 Ractor의 개수보다 작습니다. 그러므로 (대다수를 차지하는) 단일 Ractor 애플리케이션에서는 하나의 네이티브 스레드를 사용합니다.
```

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md (hunk 4 above)
Comment (riseshia) [reply ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md]: oh

File: ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md (hunk 9 above)
Comment (riseshia) [reply ko/news/_posts/2023-11-12-ruby-3-3-0-preview3-released.md]: 🙈

**Review by yous — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks!

**Pull request merged**